### ISSUE TITLE
GraphQL types with inconsistent property types

### ISSUE THREAD

**Issue by jaydensmith** [verbb/formie#192]
**Description**
The generated schema for some field types have the `containerAttributes` and `inputAttributes` properties as a String, and others as `[KeyValueType]` inconsistent with all other fields, making them hard to query. It seems to be somewhat random.

The resulting error is:
```
GraphQL error: Fields "containerAttributes" conflict because they return conflicting types [KeyValueType] and String. Use different aliases on the fields to fetch both if this was intentional.
```

```graphql
type Field_Products implements FieldInterface {
    ...
    containerAttributes: String
    inputAttributes: String
    ...
}
```

```graphql
type Field_Number implements FieldInterface {
    ...
    containerAttributes: [KeyValueType]
    inputAttributes: [KeyValueType]
    ...
}
```

**Comment by engram-design** [comment]
When you say random, is it random per query, or per field? Just looking to replicate it reliably. I might also ask to check out your full query to get it reproduced.

**Comment by jaydensmith** [comment]
@engram-design seems to be random per field, but consistent across queries.
Full query:
```graphql
{
  forms {
    id
    title
    pages {
      id
      fields {
        name
        handle
        required
        displayName
        instructions
        columnWidth
        ... on Field_Section {
          placeholder
          limitAmount
          limitType
          errorMessage
          labelPosition
          instructionsPosition
          inputAttributes {
            heading
            value
          }
          containerAttributes {
            heading
            value
          }
          cssClasses
        }
        ... on Field_Email {
          placeholder
          limitAmount
          limitType
          errorMessage
          labelPosition
          instructionsPosition
          inputAttributes {
            heading
            value
          }
          containerAttributes {
            heading
            value
          }
          cssClasses
        }
        ... on Field_Address {
          placeholder
          limitAmount
          limitType
          errorMessage
          labelPosition
          instructionsPosition
          inputAttributes {
            heading
            value
          }
          containerAttributes {
            heading
            value
          }
          cssClasses
        }
        ... on Field_Agree {
          placeholder
          limitAmount
          limitType
          errorMessage
          labelPosition
          instructionsPosition
          inputAttributes {
            heading
            value
          }
          containerAttributes {
            heading
            value
          }
          cssClasses
        }
        ... on Field_Categories {
          placeholder
          limitAmount
          limitType
          errorMessage
          labelPosition
          instructionsPosition
          inputAttributes {
            heading
            value
          }
          containerAttributes {
            heading
            value
          }
          cssClasses
        }
        ... on Field_Checkboxes {
          placeholder
          limitAmount
          limitType
          errorMessage
          labelPosition
          instructionsPosition
          inputAttributes {
            heading
            value
          }
          containerAttributes {
            heading
            value
          }
          cssClasses
        }
        ... on Field_Date {
          placeholder
          limitAmount
          limitType
          errorMessage
          labelPosition
          instructionsPosition
          inputAttributes {
            heading
            value
          }
          containerAttributes {
            heading
            value
          }
          cssClasses
        }
        ... on Field_Dropdown {
          placeholder
          limitAmount
          limitType
          errorMessage
          labelPosition
          instructionsPosition
          inputAttributes {
            heading
            value
          }
          containerAttributes {
            heading
            value
          }
          cssClasses
        }
        ... on Field_Entries {
          placeholder
          limitAmount
          limitType
          errorMessage
          labelPosition
          instructionsPosition
          inputAttributes {
            heading
            value
          }
          containerAttributes {
            heading
            value
          }
          cssClasses
        }
        ... on Field_Group {
          placeholder
          limitAmount
          limitType
          errorMessage
          labelPosition
          instructionsPosition
          inputAttributes {
            heading
            value
          }
          containerAttributes {
            heading
            value
          }
          cssClasses
        }
        ... on Field_Heading {
          placeholder
          limitAmount
          limitType
          errorMessage
          labelPosition
          instructionsPosition
          inputAttributes {
            heading
            value
          }
          containerAttributes {
            heading
            value
          }
          cssClasses
        }
        ... on Field_Hidden {
          placeholder
          limitAmount
          limitType
          errorMessage
          labelPosition
          instructionsPosition
          inputAttributes {
            heading
            value
          }
          containerAttributes {
            heading
            value
          }
          cssClasses
        }
        ... on Field_Html {
          placeholder
          limitAmount
          limitType
          errorMessage
          labelPosition
          instructionsPosition
          inputAttributes {
            heading
            value
          }
          containerAttributes {
            heading
            value
          }
          cssClasses
        }
        ... on Field_MultiLineText {
          placeholder
          limitAmount
          limitType
          errorMessage
          labelPosition
          instructionsPosition
          inputAttributes {
            heading
            value
          }
          containerAttributes {
            heading
            value
          }
          cssClasses
        }
        ... on Field_Name {
          placeholder
          limitAmount
          limitType
          errorMessage
          labelPosition
          instructionsPosition
          inputAttributes {
            heading
            value
          }
          containerAttributes {
            heading
            value
          }
          cssClasses
        }
        ... on Field_Number {
          placeholder
          limitAmount
          limitType
          errorMessage
          labelPosition
          instructionsPosition
          inputAttributes {
            heading
            value
          }
          containerAttributes {
            heading
            value
          }
          cssClasses
        }
        ... on Field_Phone {
          placeholder
          limitAmount
          limitType
          errorMessage
          labelPosition
          instructionsPosition
          inputAttributes {
            heading
            value
          }
          containerAttributes {
            heading
            value
          }
          cssClasses
        }
        ... on Field_Radio {
          placeholder
          limitAmount
          limitType
          errorMessage
          labelPosition
          instructionsPosition
          inputAttributes {
            heading
            value
          }
          containerAttributes {
            heading
            value
          }
          cssClasses
        }
        ... on Field_Recipients {
          placeholder
          limitAmount
          limitType
          errorMessage
          labelPosition
          instructionsPosition
          inputAttributes {
            heading
            value
          }
          containerAttributes {
            heading
            value
          }
          cssClasses
        }
        ... on Field_Repeater {
          placeholder
          limitAmount
          limitType
          errorMessage
          labelPosition
          instructionsPosition
          inputAttributes {
            heading
            value
          }
          containerAttributes {
            heading
            value
          }
          cssClasses
        }
        ... on Field_Table {
          placeholder
          limitAmount
          limitType
          errorMessage
          labelPosition
          instructionsPosition
          inputAttributes {
            heading
            value
          }
          containerAttributes {
            heading
            value
          }
          cssClasses
        }
        ... on Field_Tags {
          placeholder
          limitAmount
          limitType
          errorMessage
          labelPosition
          instructionsPosition
          inputAttributes {
            heading
            value
          }
          containerAttributes {
            heading
            value
          }
          cssClasses
        }
        ... on Field_Users {
          placeholder
          limitAmount
          limitType
          errorMessage
          labelPosition
          instructionsPosition
          inputAttributes {
            heading
            value
          }
          containerAttributes {
            heading
            value
          }
          cssClasses
        }
      }
      settings {
        submitButtonLabel
      }
    }
  }
}
```

**Comment by engram-design** [comment]
Should be fixed for the next release. To get the fix early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-3 as 1.2.24",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Fixed in [1.2.25](https://github.com/verbb/formie/releases/tag/1.2.25)
